accepted/future-releases/0546-patterns/feature-specification.md

```diff
@@ -952,12 +968,12 @@ It is a compile-time error if:
     to local variables, which are also the only kind of variables that can be
     declared by patterns.*
 
-*   The matched value type for a variable pattern is not assignable to the
+*   The matched value type for an identifier pattern is not assignable to the
     corresponding variable's type.
 
 *   The same variable is assigned more than once. *In other words, a pattern
-    assignment can't have multiple variable subpatterns with the same name. This
-    prohibits code like:*
+    assignment can't have multiple identifier subpatterns with the same name.
```

Comment:
```suggestion
    assignment can't have multiple variable or identifier subpatterns with the same name.
```

Reply:
With this change, pattern assignments can no longer contain variable patterns at all, only identifier patterns, so we don't need to call them out here.